tests/components/laundrify/test_coordinator.py

```diff
@@ -5,30 +5,25 @@
 from homeassistant.components.laundrify.const import DOMAIN
 from homeassistant.core import HomeAssistant
 
-from . import create_entry
+from tests.common import MockConfigEntry
 
 
-async def test_coordinator_update_success(hass: HomeAssistant) -> None:
+async def test_coordinator_update_success(
+    hass: HomeAssistant, laundrify_config_entry: MockConfigEntry
+) -> None:
     """Test the coordinator update is performed successfully."""
-    config_entry = create_entry(hass)
-    await hass.config_entries.async_setup(config_entry.entry_id)
-    await hass.async_block_till_done()
-    coordinator = hass.data[DOMAIN][config_entry.entry_id]["coordinator"]
+    coordinator = hass.data[DOMAIN][laundrify_config_entry.entry_id]["coordinator"]
```

Comment:
We should not touch the internals during tests. Let's hold off from refactoring this tests

Reply:
What do you mean by saying "_not touch the internals_"?